src/transformers/modeling_tf_longformer.py

```diff
@@ -144,23 +144,21 @@ def call(
 
         """
         # retrieve input args
-        hidden_states, attention_mask, output_attentions = inputs
-
-        attention_mask = tf.squeeze(tf.squeeze(attention_mask, axis=2), axis=1)
-        # is index masked or global attention
-
-        is_index_masked = tf.math.less(attention_mask, 0)
-        is_index_global_attn = tf.math.greater(attention_mask, 0)
```

Comment:
These operations should be done once at not be repeated for every layer -> unnecessary slow-down